docs/docs/modeling.md

```diff
@@ -114,7 +114,25 @@ classDiagram
 
 ## AssignmentSheet - 分担票
 
-## TenantsWork -
+## TenantsWork - 割当
+
+## TenantsWork(Entity)
+
+TenantsWork refers to the set of tasks assigned(AssignedCategory) to each tenant.
+
+Constraints:
+
+- cannot exist without any AssignedCategory.
+- have only one TenantId.
+- can have more than 1 AssignedCategory.
```

Comment:
It would be better to standardize the numerical notation to either `1` or `one`.